### ISSUE TITLE
Can't install or upgrade Monitoring via UI due to global tolerations

### ISSUE THREAD

**Issue by gaktive** [rancher/dashboard#4737]
Stemming from work in https://github.com/rancher/rancher/issues/35744 and https://github.com/rancher/rancher/issues/34439, the UI won't allow "rancher-monitoring-crd" to be seen since a toleration is missing. The backend addressed their side of what's going on late into 2.6.3 so the UI just has to catch up.

Per the backend, two issues that blocks us from installing the monitoring chart from the UI:

1. Even we add the toleration to the crd chart and get it installed, the rancher-monitoring app will not be installed. in other words, the installation process stops at the 1st failure on the crd chart.
2. Every time we edit the rancher-monitoring app in the UI, it will trigger an upgrade on the crd chart, but since the UI does not provide the toleration in the values it passes to Helm, the upgrade always fail and we have to edit the app manually which brings us back to the issue 1.

Because of the above 2 issues, although we have added the support for toleration in the crd chat, because of the missing UI component, we are unable to install the rancher-monitoring via UI on a cluster where customized taints are applied to nodes.

Repro:
1. Go to the apps and marketplace, install monitoring and the previously mentioned version
2. While waiting for install to process you can check whether the pods are being deployed or whether the app "rancher-monitoring-crd" is working in the "installed apps" page. 
3. Once you see it's failed, use the kebab menu of the CRD to "edit/upgrade" 
4. On the yaml page add tolerations equivalent to the previously created taints. For example: 
```
tolerations:
- key: "qa"
  operator: "Equal"
  value: "test"
  effect: "NoSchedule"
  
```
5. Click save and return to the installed apps page. 
6. View the yaml of the crd from the kebab menu and notice the previously added tolerations are not listed.

Workaround for this issue: instead of using Rancher UI, use Helm to install the rancher-monitoring chart into the cluster directly.

**Comment by gaktive** [comment]
We may have an internal reference here; confirming if SURE-4019 is a match.

**Comment by codyrancher** [comment]
This issue is currently blocked by a backend issue https://github.com/rancher/rancher/issues/38403

**Comment by gaktive** [comment]
Related Jira closed by support.

**Comment by gaktive** [comment]
Good news: https://github.com/rancher/rancher/issues/38403 is in progress. However, the related PR (https://github.com/rancher/rancher/pull/41160) is going through review still.

**Comment by gaktive** [comment]
Backend pushing their work to Q3 so we'll do the same.

**Comment by gaktive** [comment]
This will likely be a Won't Fix but need to confirm.

**Comment by codyrancher** [comment]
@gaktive do I need to confirm with anyone in particular? 

It looks like it's something we can close since we closed SURE-4019 and judging from the backend issues we're likely not going to prioritize this.

**Comment by gaktive** [comment]
@codyrancher let's close this then. If something bubbles back up, we can check with Observability.
